### PR TITLE
robust harvest heals pod people

### PR DESCRIPTION
**Pull request by boodaliboo**
# General Documentation

### Intent of your Pull Request

pod people heal from plant food
### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
pod people heal from plant food

### What should players be aware of when it comes to the changes your PR is implementing?
pod people heal from plant food
### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?
pod people heal from plant food
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

# Changelog

pod people heal from plant foodr.

:cl:  
rscadd: pod people heal from plant food

/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
pod stuff is handled in their species file for these and robust harvest already has an effect don't think this is necessary

**Comment by SomeguyManperson** [comment]
wait you just stapled this to the end of the file it won't even work

**Comment by Moggram** [comment]
Why is this not under the handle chemicals part?

**Comment by SomeguyManperson** [comment]
robust harvest already has an effect on podpeople anyways
